skimage/data/__init__.py

```diff
@@ -11,6 +11,7 @@
 from .. import data_dir
 from ..io import imread, use_plugin
 from ._binary_blobs import binary_blobs
+import xml
```

Comment:
I would not group data by their formats, but rather by their purposes.  So if you want, use `skimage.data.detect.*`.